docs/source/project_create.rst

```diff
@@ -183,16 +185,32 @@ irrelevant (or relevant).
 .. figure:: ../images/setup_prior_random_1rel.png
    :alt: ASReview prior knowledge random
 
+In the Validation mode when selecting random records, one can choose random
+records from the subset of initially labeled relevant, irrelevant or unseen
+records. The initial labels are displayed via a color-coded bar above each
+record. 
+
+.. figure:: ../images/setup_prior_knowledge_random_validate.png
+   :alt: ASReview prior knowledge selector
+
+
+Inspect
+-------
```

Comment:
This is not part of the 2 options the user interface offers. Might fit better somewhere else.

Reply:
It is part of the new user interface of the Validation mode. 
![setup_prior_knowledge_random_validate](https://github.com/asreview/asreview/assets/36502709/dde806d3-b7f5-4735-8634-0837c40c85b7)

Reply:
It is part of the new Validation mode:
![setup_prior_knowledge_random_validate](https://github.com/asreview/asreview/assets/36502709/b13bb831-b677-494c-8588-dcf30ff4e029)